pouchdb-core/pouchdb-core-tests.ts

```diff
@@ -40,9 +40,9 @@ namespace PouchDBCoreTests {
 
     function testBulkDocs() {
         const db = new PouchDB<MyModel>();
-        type MyModel = { property: 'someProperty '};
-        let model: PouchDB.Core.Document<MyModel>;
```

Comment:
This is not valid code: variable `model` is used before being assigned.
What is purpose of defining string literal on model such way in tests? Test never check that.

I it should check, if type Model can be used with PouchDB.Core.Document, its not needed. This is checked with calling method few lines below.